lib/situm_ar.dart

```diff
@@ -270,11 +272,17 @@ class _ARWidgetState extends State<ARWidget> with WidgetsBindingObserver {
       isArVisible = true;
       isMapCollapsed = false;
     });
-    Future.delayed(animationDuration, () {
+    centerAction() {
       // "Center" the MapView into the ScrollView:
-      scrollController
-          .jumpTo(scrollController.position.maxScrollExtent * 2 / 5);
-    });
+      if (isArVisible) {
+        scrollController
+            .jumpTo(scrollController.position.maxScrollExtent * 2 / 5);
+      }
+    }
+
+    Future.delayed(animationDurationWithDelay, centerAction);
+    // Watchdog:
+    Future.delayed(const Duration(seconds: 1), centerAction);
```

Comment:
Entiendo que esto se hace porque la primera vez puede fallar?

Reply:
Correcto, non é o máis elegante pero sirve para cubrir as espaldas.